service/apis/meta_service_test.go

```diff
@@ -68,3 +70,129 @@ func TestMetaService(t *testing.T) {
 	}
 	assert.Equal(t, 3, len(fs), "return fields don't match metadata define")
 }
+
+func TestDeleteMetadata(t *testing.T) {
+	dc := db.NewDGClient("127.0.0.1:9080")
```

Comment:
should the tests in this file also close the client when completed?

or perhaps it's not necessary in a testing env where maybe each test runs from a known state

Reply:
It not mandatory but close connection should be good practice for single unit test. I will add it.